client/components/ProductOverview/layouts/ProductMainInfos.js

```diff
@@ -1,31 +1,46 @@
-import React, { useContext } from 'react';
 import { ProductContext } from '../../../context/ProductContext';
-import { AddToBag } from '../widgets/AddToBag';
-import { AddToFav } from '../widgets/AddToFav';
+import React, { useContext } from 'react';
 import { CustomSelect } from '../widgets/CustomSelect';
 import { Ratings } from '../widgets/Ratings';
 import { StyleSelector } from '../widgets/StyleSelector';
+import { ToBagToFavBtns } from '../widgets/ToBagToFavBtns';
 
 export const ProductMainInfos = () => {
-  const { curProduct } = useContext(ProductContext);
-  let dOptions = ['S', 'M', 'L'];
-  let dQty = [1, 2, 3];
+  const { curProduct, curStyle } = useContext(ProductContext);
+  const curSizes = [];
+  const curQty = [];
+
+  for (let i in curStyle.skus) {
+    curSizes.push(curStyle.skus[i].size);
+    curQty.push(curStyle.skus[i].quantity);
+  }
+
   return (
     <div className="mProdMainInfo">
       <Ratings />
       <div className="fs-4 text-muted">{curProduct.category}</div>
       <div className="fs-2 fw-bold">{curProduct.name}</div>
-      <div className="fs-5">${curProduct.default_price}</div>
+      {curStyle.sale_price ? (
+        <div className="fs-5 fw-bold d-flex">
```

Comment:
if this div stays the same in both conditional cases, maybe conditionally only the inner HTML?